### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
Run rekres and bump deps.

### CODE REVIEW

**Review by smira — APPROVED** [review]

**Review by frezbo — APPROVED** [review]

File: internal/backend/runtime/omni/state_etcd.go [hunk 1]
```diff
@@ -280,7 +278,6 @@ func getExternalEtcdClient(ctx context.Context, params *config.EtcdParams, logge
 		DialOptions: []grpc.DialOption{
 			grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(constants.GRPCMaxMessageSize)),
 			grpc.WithSharedWriteBuffer(true),
-			experimental.WithRecvBufferPool(grpc.NewSharedBufferPool()),
```
[the remark this answers is not in the document]
Comment (DmitriyMV) [reply internal/backend/runtime/omni/state_etcd.go:283]: Per [docs](https://pkg.go.dev/google.golang.org/grpc/experimental#BufferPool)

> BufferPool returns a grpc.ServerOption that configures the server to use the provided buffer pool for parsing incoming messages and for temporary buffers when marshaling outgoing messages. By default, mem.DefaultBufferPool is used, and this option only exists to provide alternative buffer pool implementations to the server, such as more optimized size allocations etc. However, the default buffer pool is already tuned to account for many different use-cases.

So we should be good.

[the remark this answers is not in the document]
Comment (smira) [reply internal/backend/runtime/omni/state_etcd.go:283]: This is a server one, and this one is a client?

[the remark this answers is not in the document]
Comment (DmitriyMV) [reply internal/backend/runtime/omni/state_etcd.go:283]: Both [BufferPool](https://pkg.go.dev/google.golang.org/grpc/experimental#BufferPool) (for server) and [WithBufferPool](https://pkg.go.dev/google.golang.org/grpc/experimental#WithBufferPool) (for client) have this doc.

[the remark this answers is not in the document]
Comment (smira) [reply internal/backend/runtime/omni/state_etcd.go:283]: ok, sounds good

**Pull request merged**